pytest.ini

```diff
@@ -5,6 +5,8 @@ markers =
     multiclass
     wandb_args
     flaky: tests marked as flaky need to be debugged
+    server_issues: tests that fail against the mock server with connection errors
+    slow: unit tests that take more than 5 seconds to run
```

Comment:
we should be able to collect statistics from CI execution, maybe we can be more methodical about how we label tests as slow. I guess there are some pytest extensions that have interesting solution for this...

Reply:
https://app.circleci.com/insights/github/wandb/wandb/workflows/main/tests?branch=main shows the 100 slowest tests at the bottom, so we already have those measured. Actually, if you'd like me to just rework this PR and only mark the slow ones as determined by CircleCI, I can do that. Most of those aren't unit tests of course, so even the 100-th slowest test takes 44s, so I wonder if I can find tests all the way down to 15 seconds